app/controllers/events_controller.rb

```diff
@@ -9,7 +9,7 @@ def show
 
   def random_attendee
     attendes = meetup.attendence(urlname, event_id)
-    person = attendes.shuffle.pop[:member]
+    person = attendes.sample.member
```

Comment:
What about removing this logic from the controller? Something like creating a `random_atendee` method on the service.

I also think we're gonna need to make a request to `/members/:member_id` to get a standardized profile image, as shown [here](https://www.meetup.com/pt-BR/meetup_api/docs/2/member/?uri=%2Fmeetup_api%2Fdocs%2F2%2Fmember%2F#get).